### DISCUSSION TITLE
Non-intuitive method to configure "Format for time, date and currency"

### DISCUSSION THREAD

**Discussion by dalers** [kimai/kimai#4742] [General]
Hi all, I'm just getting started with Kimai so my apologies if I'm missing something. I'm in Canada, use English and prefer ISO 8601 dates (yyyy-mm-dd) and 24hr time for clarity, convenience for not needing to enter AM/PM in a time (and I never remember how midnight and noon are represented), and correct sorting should I need to export records into a spreadsheet for processing of some type.

I had expected date and time format to be individually configurable, but instead found they were determined by a Language selector, and I had to search through the supported languages for my desired time and date formats (selecting "English (Sweden)", and happily finding it used a comma thousands separator and period decimal point which are the standard here). 

It seems it would be much faster to individually select my desired date format, time formats, thousands separator and decimal point indicator, e.g. four selections instead of examining 105 English language entries. Am I missing something? Is there an advantage of the method used?

**Comment by kevinpapst** [comment]
Don't get me wrong, I appreciate your input, but...

- Following standards is better than cooking your own rules.
- It is much simpler for the developer.
- At some point it might be changed, but there have been enough discussions about that topic, use the search to find them.

**Comment by dalers** [comment]
Thanks for your fast response @kevinpapst  :-)

I fully agree following standards should be preferred over arbitrary personal preferences, but there is no "standard" that English-speaking Canadians write dates in yyyy-mm-dd format and time in 12-hr format (with "a.m." and "p.m." in lower-case with periods), or that French-speaking Canadians write time in 24-hr format using " h " for the hour/minute separator and currency with a space for the thousands separator.  

I searched for "time" in the discussions, and read through 14 pages of discussion titles, and a half-dozen topics in their entirely, expecting to find some discussion on the topic and was surprised not to find anything relevant. Perhaps I inadvertently didn't search _all_ topics, but I did try.

Cheers,
Dale
